sphinx_click/ext.py

```diff
@@ -7,7 +7,7 @@
 from sphinx.util import logging
 
 LOG = logging.getLogger(__name__)
-CLICK_VERSION = tuple(int(x) for x in click.__version__.split('.'))
+CLICK_VERSION = tuple(int(x) for x in click.__version__.split('.')[0:2])
```

Comment:
I'm unsure what this is about. Could you elaborate?

Reply:
TLDR; In semantic versioning (semver.org) the prerelease identifiers are separated by a '-' acan be non-numerical. For example, 2.2.0-alpha will crash (`"0-alpha"` is not `int()`'able, it will die), and it's easy to drop the patch level anyhow because it's not used in determining anything later on.  

I really didn't think something like `click.__version__.split(|d| d == '.' || d == '-')` was necessary here

Reply:
Ah, indeed. Sorry, I missed the commit message detailing that